Libraries/Network/RCTNetworking.m

```diff
@@ -190,6 +190,19 @@ @implementation RCTNetworking
   return nil;
 }
 
+- (NSDictionary<NSString *, id> *)stripNullsInRequestHeaders:(NSDictionary<NSString *, id> *)headers
+{
+  NSMutableDictionary *result = [NSMutableDictionary dictionaryWithCapacity:[headers count]];
```

Comment:
Done. I'm assuming `headers[key] = val` doesn't work as a replacement for `setObject:forKey:`? I'm really not much of an ObjC person...